device/alpaka/src/seeding/track_params_estimation.cpp

```diff
@@ -0,0 +1,77 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+//
+// Local include(s).
+#include "traccc/alpaka/utils/definitions.hpp"
+
+// Project include(s).
+#include "traccc/alpaka/seeding/track_params_estimation.hpp"
+#include "traccc/alpaka/utils/definitions.hpp"
+#include "traccc/seeding/device/estimate_track_params.hpp"
+
+namespace traccc::alpaka {
+
+struct EstimateTrackParamsKernel {
+    template <typename TAcc>
+    ALPAKA_FN_ACC void operator()(
+        TAcc const& acc,
+        spacepoint_collection_types::const_view spacepoints_view,
+        seed_collection_types::const_view seed_view,
+        cell_module_collection_types::const_view modules_view,
+        const vector3 bfield,
+        const std::array<traccc::scalar, traccc::e_bound_size> stddev,
+        bound_track_parameters_collection_types::view params_view) const {
+        auto const globalThreadIdx =
+            ::alpaka::getIdx<::alpaka::Grid, ::alpaka::Threads>(acc)[0u];
+        device::estimate_track_params(globalThreadIdx, spacepoints_view,
+                                      seed_view, modules_view, bfield, stddev,
+                                      params_view);
+    }
+};
```

Comment:
It is **not** for this PR, but I wonder... Do we need to define these separate structs? Is Alpaka not able to use lambdas instead? (I genuinely don't know...)

Reply:
There is some nuance with Kernel Definitions: https://alpaka.readthedocs.io/en/latest/basic/library.html#kernel-definition

For that reason, and because it more closely mimics the layout of the CUDA code, I went with the struct layout for now.
I'm also not sure how the later dynamic shared memory (`extern __shared__`) equivalent would work, which is defined using traits, with an example in the `seed_finding.cpp` code (though

Reply:
Okay, let's just go with the structs for now. :wink: